tower-balance/src/weight.rs

```diff
@@ -0,0 +1,167 @@
+use futures::{Async, Poll};
+use std::ops;
+use tower_discover::{Change, Discover};
+use tower_service::Service;
+
+use Load;
+
+/// A weight on [0.0, ∞].
```

Comment:
It looks like `Weight::MIN` is `0.0`. What happens then? With the `impl Div<Weight> for Cost` and `Count`, it looks like that would yield `NAN`. What does that mean? (I hate IEEE 754 for introducing NAN....)

Reply:
We treat this as `INFINITY`